Oostdijk, Aaron/Week 1/HotspotPositionCalc.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;

```

Comment:
Are the System library's needed if you don't use them?

Reply:
Shouldn't be. Unity's standard template adds this, because a lot of the time people will want to use collections of some kind, but I tend to stick to generic (list/dict) or standard arrays ([])